scripts/asm-installer/asm_vm

```diff
@@ -1110,6 +1110,23 @@ this project, run:
   --project="${PROJECT_ID}"
 EOF
     fi
+
+    info "Verifying GCE image..."
+    local SOURCE_IMAGE_PATH="$(echo "$(retry 2 run gcloud compute instance-templates \
+      describe "${SOURCE_INSTANCE_TEMPLATE}" \
+      --format=json \
+      --project="${PROJECT_ID}")" \
+    | jq '.properties.disks[] | select(.initializeParams.sourceImage != null) | .initializeParams.sourceImage')"
+
+
+    if [[ "$SOURCE_IMAGE_PATH" == *projects/*-cloud/global/images*  ]]; then
```

Comment:
Should the regex be `^projects/*-cloud/global/images`?

Reply:
yes I can change this source image path to start with "projects" string

Reply:
Not sure why you see the full address. I got
```
disks:
  - autoDelete: true
    boot: true
    deviceName: centos
    index: 0
    initializeParams:
      diskSizeGb: '20'
      diskType: pd-balanced
      sourceImage: projects/centos-cloud/global/images/centos-8-v20210316
```

Are there any differences in how you created both instance templates?

Reply:
I just use the UI on google cloud to create it. As this both path exists on google gcloud platform we should support it both

Reply:
SGTM.